library/Icingadb/Common/DetailActions.php

```diff
@@ -132,8 +132,11 @@ public function addMultiselectFilterAttribute(BaseHtmlElement $element, Filter\R
      */
     public function addDetailFilterAttribute(BaseHtmlElement $element, Filter\Rule $filter): self
     {
+        if (! $this->getDetailActionsDisabled()) {
+            $element->getAttributes()->set('data-action-item', true);
+        }
```

Comment:
I thought you said you'll apply this the same as the other attribute? Where's the callback then?